src/code/templates/clutch-game.js

```diff
@@ -143,15 +142,21 @@ export default class ClutchGame extends Component {
     const parentGenomeClass = classNames('parent');
 
     function parentGenomeView(sex) {
+      let parentChangeableGenes;
+      // Changeable genes are either of the form ["wings, legs"] or [{mother: "wings", father:""}, {mother: "", father: "wings"]
```

Comment:
I put everything in a new commit on the branch, so it should be easier to review